src/Maintenance/Maintenance/Az.Maintenance.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.0'
+ModuleVersion = '1.2.0'
```

Comment:
this psd1 file should not be updated. It will be updated automatically before release. Please revert your change.

Reply:
Fixed